### PR TITLE
Add fixed width for Content aside

### PR DESCRIPTION
**Pull request by viktoria-mahmud**

### CODE REVIEW

File: assets/sass/patterns/organisms/content-aside.scss
Comment (discodavey) [inline assets/sass/patterns/organisms/content-aside.scss:113]: Can we please change this to max-width

Comment (viktoria-mahmud) [reply assets/sass/patterns/organisms/content-aside.scss]: Done.

**Review by discodavey — APPROVED** [review]
All good to go once @chris-huggins has approved and is happy with it

**Pull request merged**
